model/src/gateway/event/kind.rs

```diff
@@ -12,6 +12,8 @@ pub enum EventType {
     ChannelDelete,
     ChannelPinsUpdate,
     ChannelUpdate,
+    #[serde(rename = "APPLICATION_COMMAND_PERMISSIONS_UPDATE")]
+    CommandPermissionsUpdate,
```

Comment:
why not name it as it is?

Reply:
The name is quite long, I chose to shorten as it's already the case for some events.